utils/ratlib.py

```diff
@@ -18,7 +18,7 @@
 
 MIRC_CONTROL_CODES = ["\x0F", "\x16", "\x1D", "\x1F", "\x02",
                       "\x03([1-9][0-6]?)?,?([1-9][0-6]?)?"]
-STRIPPED_CHARS = ';\''
+STRIPPED_CHARS = ';'
```

Comment:
???

Reply:
No further reason to strip quotes or semi-colons, as we're not vulnerable to this type of injection due to composed strings.